admin-dev/filemanager/execute.php

```diff
@@ -1,12 +1,14 @@
 <?php
 include('config/config.php');
 if ($_SESSION['verify'] != 'RESPONSIVEfilemanager') {
-    die('forbiden');
+    die('Forbidden');
 }
 include('include/utils.php');
 
-$_POST['path_thumb'] = $thumbs_base_path.$_POST['path_thumb'];
-if (!isset($_POST['path_thumb']) && trim($_POST['path_thumb']) == '') {
+$_POST['path'] = isset($_POST['path']) ? str_replace("\0", '', $_POST['path']) : null;
+$_POST['path_thumb'] = isset($_POST['path_thumb']) ? $thumbs_base_path . str_replace("\0", '', $_POST['path_thumb']) : null;
```

Comment:
Why choosing `null` and not an empty string?

Reply:
Just because if `$_POST['thisIsAKey']` doesn't exists, it's logically null, but I can change if you want.